### PR TITLE
[added] wrapperProps and wrapperStyle props

### PR DESCRIPTION
**Pull request by Neitsch**
Rebased PR 50.

### CODE REVIEW

File: lib/Autocomplete.js [hunk 1]
```diff
@@ -13,13 +13,19 @@ let Autocomplete = React.createClass({
     shouldItemRender: React.PropTypes.func,
     renderItem: React.PropTypes.func.isRequired,
     menuStyle: React.PropTypes.object,
-    inputProps: React.PropTypes.object,
-    labelText: React.PropTypes.string
+    labelText: React.PropTypes.string,
+    wrapperProps: React.PropTypes.object,
+    wrapperStyle: React.PropTypes.object,
+    inputProps: React.PropTypes.object
```
Comment (CMTegner) [inline lib/Autocomplete.js]: Why was `inputProps` moved?

File: lib/Autocomplete.js [hunk 2]
```diff
@@ -329,4 +335,3 @@ let Autocomplete = React.createClass({
 })
 
 module.exports = Autocomplete
-
```
Comment (CMTegner) [inline lib/Autocomplete.js]: Try to reduce commit noise by omitting changes to whitespace.

File: lib/Autocomplete.js [hunk 3]
```diff
@@ -298,8 +304,8 @@ let Autocomplete = React.createClass({
         state: this.state
       })
     }
-    return ( 
-      <div style={{display: 'inline-block'}}>
+    return (
+      <div {...this.props.wrapperProps} style={{...this.props.wrapperStyle}}>
```
Comment (CMTegner) [inline lib/Autocomplete.js]: Perhaps `wrapperProps` should come last so the consumer can optionally specify `style` in `wrapperProps`?

e.g. `<Autocomplete wrapperProps={{ style: { ... }, ... }} />`

**Comment by Neitsch** [comment]
Thanks for the advice! Hope this works better now.

**Comment by CMTegner** [comment]
LGTM

**Pull request merged**